backend/private-graph/graph/schema.graphqls

```diff
@@ -647,6 +647,14 @@ type ErrorObjectConnection implements Connection {
 	pageInfo: PageInfo!
 }
 
+type Service {
+	id: ID!
+	projectID: ID!
+	name: String!
+	status: String!
```

Comment:
Not blocking but it would be great if this was an enum so we can map the data more cleanly later on.

Reply:
I agree! I tried this and hit a snag with something in the private-graph but will give it a look